docs/core/development/policy/release-process.rst

```diff
@@ -184,31 +184,12 @@ Prepare the branch
 #. Commit the version and ``README.rst`` changes.
```

Comment:
I think the previous info about the Quotes should be removed.

Also, I think that instead of reverting the pre-release changes, we can just manually update the release date and version inside the NEWS file

Reply:
sure, I don't want to change the release process here around changelogs/RC/final in this PR